hibernate-core/src/main/java/org/hibernate/Hibernate.java

```diff
@@ -259,6 +259,26 @@ public static <T> Class<? extends T> getClass(T proxy) {
 		return (Class<? extends T>) result;
 	}
 
+	/**
+	 * Get the true, underlying class of a proxied entity. This operation might
+	 * initialize a proxy by side effect.
```

Comment:
At the moment, the Javadoc between this method and `#getClass` are identical - how would users know the difference?

```suggestion
	 * Get the true, underlying class of a proxied entity. 
	 * <p/>
	 * Like {@link #getClass}, this operation might initialize a proxy by side effect.
	 * However, here the initialization is avoided if possible.  If the entity type is 
	 * defined with subclasses, the proxy will need to be initialized to properly
	 * determine the class.
```